### PR TITLE
Implemented glob pattern support for 'runner.src' method (close #980)

### LINKED ISSUE #980: Glob patterns in runner.src method
### Are you requesting a feature or reporting a bug?

feature

### What is the current behavior?

```runner.src(...)``` method does not support glob patterns in parameters, it's inconsistent with [cli.](https://github.com/DevExpress/testcafe/blob/c4c6989b97e78a82340199daa8bb322f8ebb1775/docs/articles/documentation/using-testcafe/command-line-interface.md#file-pathglob-pattern)

### What is the expected behavior?

I want to have possibility specify global pattern or array of patterns in ```runner.src(...)``` method.
```javascript
runner.src(['C:/project/subProject1/**/*.js', 'C:/project/subProject2/**/*.js']);
```

### Specify your

* operating system: any
* testcafe version: 0.10.0
* node.js version: any

### PR DESCRIPTION
**Pull request by miherlosev**
@kirovboris @AndreyBelym @churkin

### CODE REVIEW

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
FPR

File: src/runner/index.js [hunk 1]
```diff
@@ -156,7 +155,13 @@ export default class Runner extends EventEmitter {
...
 
     src (...sources) {
-        sources = flatten(sources).map(path => resolvePath(path));
+        // NOTE: the behavior here is consistent with the CLI
+        // src([]) won't search for tests in default folders, while
+        // src() will.
+        sources = arguments.length === 0 ? void 0 : flatten(sources);
+
+        if (!sources || sources.length)
+            sources = parseFileList(sources, process.cwd());
```
Comment (AndreyBelym) [inline src/runner/index.js]: We can't add default test folders here. Take a look at the line below:
```js
    this.bootstrapper.sources = this.bootstrapper.sources.concat(sources);
```

It means you can chain `src()` calls:
```js
    .src()
    .src()
    .src()
```
If we handle default folders in the `src` method, they will be added 3 times to the sources. 

So we should move the logic for handling default folders to the Bootstrapper class.

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
FPR

**Pull request merged**